k8s/demo/galera-xtradb-cluster/deployments/README.md

```diff
@@ -0,0 +1,265 @@
+[README](https://github.com/openebs/openebs/tree/master/k8s/demo/mongodb).
+
+## Deploy the Percona Galera cluster with OpenEBS storage
+
+The deployment specification YAMLs are available at OpenEBS/k8s/demo/galera-xtradb-cluster/deployments. 
+Execute the following commands in order : 
+
+```
+test@Master:~/openebs$ cd k8s/demo/galera-xtradb-cluster/
+test@Master:~/openebs/k8s/demo/galera-xtradb-cluster$ ls -ltr
+total 16
+-rw-rw-r-- 1 test test 1802 Oct 30 17:44 pxc-node3.yaml
+-rw-rw-r-- 1 test test 1802 Oct 30 17:44 pxc-node2.yaml
+-rw-rw-r-- 1 test test 1797 Oct 30 17:44 pxc-node1.yaml
+-rw-rw-r-- 1 test test  174 Oct 30 17:44 pxc-cluster-service.yaml
+```
+
+```
+test@Master:~/openebs/k8s/demo/galera-xtradb-cluster$ kubectl apply -f pxc-cluster-service.yaml
+service "pxc-cluster" created
+testk@Master:~/openebs/k8s/demo/galera-xtradb-cluster$
+```
+
+```
+testk@Master:~/openebs/k8s/demo/galera-xtradb-cluster$ kubectl apply -f pxc-node1.yaml
+service "pxc-node1" created
+deployment "pxc-node1" created
+persistentvolumeclaim "datadir-claim-1" created
+```
+
+Wait till the pxc-node1 YAML is processed, after which repeat the step with pxc-node2 and pxc-node3 YAMLs. 
+
+Verify that all the replicas are up and running : 
+
+```
+test@Master:~/galera-deployment$ kubectl get pods
+NAME                                                             READY     STATUS    RESTARTS   AGE
+maya-apiserver-2245240594-r7mj7                                  1/1       Running   0          2d
+openebs-provisioner-4230626287-nr6h4                             1/1       Running   0          2d
+pvc-235b15a5-bd1f-11e7-9be8-000c298ff5fc-ctrl-2525793473-nv88z   1/1       Running   0          12m
+pvc-235b15a5-bd1f-11e7-9be8-000c298ff5fc-rep-144100677-2rm8b     1/1       Running   0          12m
+pvc-235b15a5-bd1f-11e7-9be8-000c298ff5fc-rep-144100677-gmn51     1/1       Running   0          12m
+pvc-82885a9c-bd1e-11e7-9be8-000c298ff5fc-ctrl-2555717164-sspqn   1/1       Running   0          16m
+pvc-82885a9c-bd1e-11e7-9be8-000c298ff5fc-rep-3501200001-p9778    1/1       Running   0          16m
+pvc-82885a9c-bd1e-11e7-9be8-000c298ff5fc-rep-3501200001-x3nxs    1/1       Running   0          16m
+pvc-cc94c5eb-bd1f-11e7-9be8-000c298ff5fc-ctrl-15702123-tn460     1/1       Running   0          7m
+pvc-cc94c5eb-bd1f-11e7-9be8-000c298ff5fc-rep-4137665767-0lhjb    1/1       Running   0          7m
+pvc-cc94c5eb-bd1f-11e7-9be8-000c298ff5fc-rep-4137665767-h8r6j    1/1       Running   0          7m
+pxc-node1-2984138107-zjf22                                       1/1       Running   0          16m
+pxc-node2-1007987438-q831l                                       1/1       Running   0          12m
+pxc-node3-82203929-mh5p9                                         1/1       Running   0          7m
+```
+
+## Deployment Guidelines
+
+- The galera cluster is recommended to be created with at least 3 nodes/replicas. See here for details :
```

Comment:
OpenEBS recommends creating the galera cluster with at least ...
See the following for details: